docs/concepts/bounded-context.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Bounded Context
+headline: Bounded Context
+bodyclass: docs
+layout: docs
+sidenav: doc-side-concepts-nav.html
+type: markdown
+---
+<h2 class="top">Bounded Context</h2> 
+
+Bounded Context is an autonomous component with its own domain model and its own Ubiquitous Language. 
+Large systems usually have multiple Bounded Contexts. 
+
+For example, you can consider orders, user management, shipping as examples of the separate contexts within online retail system. 
```

Comment:
Please capitalize names of Bounded Contexts: `Orders`, `User Management`, `Shipping`.